packages/dcos-integration-test/extra/test_composition.py

```diff
@@ -14,7 +15,19 @@
 
 @pytest.mark.first
 def test_dcos_cluster_is_up(dcos_api_session):
-    pass
+    def _docker_info(component):
+        # sudo is required for non-coreOS installs
+        return subprocess.check_output(['sudo', 'docker', 'version', '-f', component]).decode('utf-8').rstrip()
```

Comment:
Does `sudo` fail on core-os, or if the test is ran docker-in-docker where there isn't a sudo command?

Reply:
Yup, `sudo` [still works](https://teamcity.mesosphere.io/viewLog.html?buildId=812465&buildTypeId=DcOs_Open_Test_IntegrationTest_AwsCloudFormationSimple&tab=buildLog&_focus=127#_state=112,127) on coreOS; without `sudo` [CentOS](https://teamcity.mesosphere.io/viewLog.html?buildId=812411&buildTypeId=DcOs_Open_Test_IntegrationTest_AwsOnpremWStaticBackend&tab=buildLog&_focus=327) and [Ubuntu](https://teamcity.mesosphere.io/viewLog.html?buildId=812319&buildTypeId=DcOs_Open_Test_IntegrationTest_AzureArm&tab=buildLog&_focus=139) were failing to run `docker version` unprivileged.

Reply:
> if the test is ran docker-in-docker

I don't think that DinD is a scenario for the test (`test_dcos_cluster_is_up`) in question?

Reply:
Is DinD used when these tests are run with dcos-docker?

Reply:
It's not run as a part of `test/docker/smoke`, but is working in `test/docker` [here](https://teamcity.mesosphere.io/viewLog.html?buildId=812463&buildTypeId=DcosIo_Dcos_DockerIntegrationTests_IntegrationTestDcosDockerPr&tab=buildLog&_focus=915#_state=163).